static/app/views/dashboards/layoutUtils.tsx

```diff
@@ -184,24 +180,6 @@ export function assignDefaultLayout<T extends Pick<Widget, 'displayType' | 'layo
   return newWidgets;
 }
 
-export function enforceWidgetHeightValues(widget: Widget): Widget {
```

Comment:
This should actually be used. It was meant to enforce a minimum height because big number widgets could be 1 unit tall, whereas all of the other charts should be at least 2 units tall. So switching a chart from a big number to another chart was meant to resize to 2 units tall.

Since this was in a method that was not used anymore, and knip is currently complaining about it, I've decided to remove it for now and we will have to fix it in a follow up: DAIN-956